### ISSUE TITLE
Define component tokens for the selection controls

### ISSUE THREAD

**Issue by benteljones** [equinor/design-system#3187]

**Comment by benteljones** [comment]
The selection controls lack some defined color tokens, and have some few deviations from the standard.

**Comment by vnys** [comment]
> The selection controls lack some defined color tokens, and have some few deviations from the standard.

We can’t have more deviations 😅 I’m already starting to think we should have made semantic tokens for the button, text-input, table, chip, banner, and card. If not we just have a colour palette and not a system 🤷

**Comment by benteljones** [comment]
Agreed, we have to look at how we can avoid deviations. On this one, there is probably more talk of an extra step that we don't have. But it is quite possible that we can solve it without it @vnys

**Comment by jaakson** [comment]
I see that this was never started/concluded. If I understand "selection controls" correctly, @benteljones: @BirteThornquist is currently investigating whether the various prominence levels we have for `interactive` components are sufficient to cover all of those controls. So not just the toggle button, tabs, etc. but also the checkbox, radio buttons, switch, and so on.

Insofar as that is what she is doing right now, I will assign this issue to her and put it in progress.
